lib/livebook_web/live/session_live/persistence_live.ex

```diff
@@ -116,11 +117,17 @@ defmodule LivebookWeb.SessionLive.PersistenceLive do
                     tabindex="-1">
                     Cancel
                   </button>
-                  <button class="button-base button-blue"
-                    phx-click="confirm_file"
-                    tabindex="-1">
```

Comment:
To be clear, what I meant is literally `disabled={FileSystem.File.dir?(@draft_file)}`. We don't have to optimise the dir check either, because it only checks the path.

Reply:
Ah okay, I misunderstood "attribute" for assigns. Correcting.